src/main/java/RedBot/HParser.java

```diff
@@ -39,6 +39,15 @@ public HParser(String url) {
         this.source = source;
         //System.out.println(this.source);
     }
+
+    public static void ParserHeaded(String url) { //Parses a webpage with a user agent
+        try {
+            String doc = Jsoup.connect(url).userAgent("Mozilla/5.0 (Windows NT 10.0; Win64; x64").get().toString();
+            HParser.source = doc;
```

Comment:
If referring to a class's own instance variable, you can use `this`, as in `this.source` (refer to line 39).

Reply:
I'm aware but shouldn't we refer to static objects statically?

Reply:
I suppose you have a point, I just think it's a bit silly.